TramsDataApi/Factories/Concerns/Decisions/CreateDecisionResponseFactory.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using TramsDataApi.ResponseModels.Concerns.Decisions;
+
+namespace TramsDataApi.Factories.Concerns.Decisions
+{
+    [Obsolete("This is planned to be moved into the Concerns Casework API. If it is accessed by other APIs, please let the Concerns team know.")]
+    public class CreateDecisionResponseFactory : ICreateDecisionResponseFactory
+    {
+        public CreateDecisionResponse Create(int concernsCaseUrn, int decisionId)
+        {
+            _ = concernsCaseUrn <= 0 ? throw new ArgumentOutOfRangeException(nameof(concernsCaseUrn)) : concernsCaseUrn;
+            _ = decisionId <= 0 ? throw new ArgumentOutOfRangeException(nameof(decisionId)) : decisionId;
+
+
+            return new CreateDecisionResponse(concernsCaseUrn, decisionId);
```

Comment:
So we have validation in the CreateDecisionResponse constructor as well, which is very similar but slightly different (it has a specific error message for example). Not one to block the PR, but it would be interesting at some point to look at our approach to validation & document a pattern.

Reply:
Will look at it again when I do the next peace of work